configs/socket.js

```diff
@@ -64,5 +63,30 @@ module.exports = function createSocket(server, app) {
         top,
       });
     });
+
+    socket.on("memo/delete", async (memoId) => {
+      socket.to(socket.roomId).emit("memo/delete", memoId);
+      memoRoomDetailService.deleteMemo({
+        memoroomId: socket.roomId,
+        memoId,
+      });
```

Comment:
```suggestion
      await memoRoomDetailService.deleteMemo({
        memoroomId: socket.roomId,
        memoId,
      });
```
await이 빠진것 같아요 ㅎㅎ 수정 부탁드릴게요 ㅎㅎ  
아래 다른 비즈니스 로직도 수정 부탁드립니다 ㅎㅎ

Reply:
수정했습니다!